recipe/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "anaconda-packaging-utils" %}
+{% set version = "v0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/anaconda-distribution/anaconda-packaging-utils/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6a440a85ca29452c2900f272c971b505c56569a955be188d660dc31d9fdf044c
+
+build:
+  number: 0
+  noarch: python
+  script: pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python >=3.11
+    - conda
+    - jsonschema
+    - types-jsonschema
+    - percy >=0.1.1
+    - PyGithub
+    - pyyaml
+    - types-pyyaml
+    - requests
+    - types-requests
+
+test:
+  imports:
+    - anaconda_packaging_utils
+  source_files:
+    - anaconda-packaging-utils-config-template.yaml  # Used for self-test
+    - anaconda_packaging_utils/tests/
```

Comment:
This will already be included since setuptools (or in fact any other python packaging tool) knows that these are tests, which means that it's considered as any standard module inside `anaconda_packaging_utils`, which means it'll always be packaged up unless explicitly told not to.

Reply:
This suggestion also isn't working for me. Is it that big of a deal that this recipe can't be left as-is?